x-pack/plugins/triggers_actions_ui/public/application/sections/actions_connectors_list/components/actions_connectors_list.tsx

```diff
@@ -153,6 +162,15 @@ const ActionsConnectorsList: React.FunctionComponent = () => {
         .sort((a, b) => a.name.localeCompare(b.name))
     : [];
 
+  function setDeleteConnectorWarning(connectors: string[]) {
+    const show =
+      sumBy(connectors, (c) => {
```

Comment:
nit: I think you can just use the built in [Array.some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) function to test for this condition. I think in general lodash functions are slower and it doesn't seem like this one provides too much additional benefit.

Reply:
Resolved in this [commit](https://github.com/elastic/kibana/pull/137181/commits/dac37a909ba1e0cb375cd2635557acd48219c868)